src/state/farms/fetchPublicFarmData.ts

```diff
@@ -7,115 +7,103 @@ import multicall from 'utils/multicall'
 import { Farm, SerializedBigNumber } from '../types'
 
 type PublicFarmData = {
-  tokenAmountMc?: SerializedBigNumber
-  quoteTokenAmountMc?: SerializedBigNumber
-  tokenAmountTotal?: SerializedBigNumber
-  quoteTokenAmountTotal?: SerializedBigNumber
-  lpTotalInQuoteToken?: SerializedBigNumber
-  lpTotalSupply?: SerializedBigNumber
-  tokenPriceVsQuote?: SerializedBigNumber
-  poolWeight?: SerializedBigNumber
-  multiplier?: string
+  tokenAmountMc: SerializedBigNumber
+  quoteTokenAmountMc: SerializedBigNumber
+  tokenAmountTotal: SerializedBigNumber
+  quoteTokenAmountTotal: SerializedBigNumber
+  lpTotalInQuoteToken: SerializedBigNumber
+  lpTotalSupply: SerializedBigNumber
+  tokenPriceVsQuote: SerializedBigNumber
+  poolWeight: SerializedBigNumber
+  multiplier: string
 }
 
 const fetchFarm = async (farm: Farm): Promise<PublicFarmData> => {
   const { pid, lpAddresses, token, quoteToken } = farm
   const lpAddress = getAddress(lpAddresses)
-  const farmFetch = async () => {
```

Comment:
Hyperactive diff because spacing within the function has changed. 

THIS line declaring the async `farmFetch` was deleted